docs/Development/ModelDevelopment.md

```diff
@@ -0,0 +1,29 @@
+# Open Source
+
+We encourage any users who have modified (or would like to modify) RBM, either to fix bugs or develop new features, to contact us and coordinate development work with us. The RBM model source code is archived in Git and is publicly available through [GitHub](https://github.com). To access the source code, visit [GitHub](https://github.com), create an account, and visit [github.com/UW-Hydro/RBM](https://github.com/UW-Hydro/RBM). The routing model will soon be added to the archive as well.
+
+RBM is an open source development model and is released under the terms of the [GNU General Public License v2 (GPL).](http://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
```

Comment:
Yifan - did you put this in here or did I? If I did - I'm not sure that is the correct open source license reference - and you should double check.